flink-connector-debezium/src/main/java/com/alibaba/ververica/cdc/debezium/internal/DebeziumOffsetSerializer.java

```diff
@@ -38,6 +39,7 @@
 
     public DebeziumOffset deserialize(byte[] bytes) throws IOException {
         ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.enable(DeserializationFeature.USE_LONG_FOR_INTS);
```

Comment:
Why need this?

Reply:
I have added  an issue #121 to explain this. You can take a look.

Reply:
This commit may need to be put in another pull request since it is for another issue. when I added this pull request, I just had one commit ```a27b810``` for it. I don't why this commit ```66a5a92``` appears here.